build.rs

```diff
@@ -43,12 +43,14 @@ fn main() -> anyhow::Result<()> {
                 "simd",
                 strategy,
             )?;
-            write_testsuite_tests(
-                &mut out,
-                "tests/spec_testsuite/proposals/simd/simd_const.wast",
-                "simd",
-                strategy,
-            )?;
+            // FIXME this uses some features from the reference types proposal
+            // (multi-table) which aren't fully implemented yet
+            // write_testsuite_tests(
+            //     &mut out,
+            //     "tests/spec_testsuite/proposals/simd/simd_const.wast",
+            //     "simd",
+            //     strategy,
+            // )?;
```

Comment:
Does it use reference types in the checked in version of the testsuite or in the HEAD version of testsuite? I don't understand how this PR requires ignoring this test since it is currently passing in CI and I can't find anything below that would cause it to fail.

Reply:
Ah this has to do with the bugfix in this PR where we weren't actually running this entire test previously. Before this PR we would accidentally stop running tests [here](https://github.com/WebAssembly/testsuite/blob/ab59e3a7cb039f9a50d4e24da7634dd4dd472ce3/proposals/simd/simd_const.wast#L129-L132) which means we never actually ran [this test](https://github.com/WebAssembly/testsuite/blob/ab59e3a7cb039f9a50d4e24da7634dd4dd472ce3/proposals/simd/simd_const.wast#L885-L969) which fails due to its [usage of `funcref`](https://github.com/WebAssembly/testsuite/blob/ab59e3a7cb039f9a50d4e24da7634dd4dd472ce3/proposals/simd/simd_const.wast#L912) and how the binary encoding for that has been in flux recently and not all the tools agree.

I believe we can re-enable the test once wasmparser and `wast` are updated in wasmtime

Reply:
Nice, good find!